docs/source/learn/eager.md

```diff
@@ -24,24 +24,40 @@ res = model(input)
 torch_xla.sync()
 ```
 
-The actual model compilation and device execution happens when
-`torch_xla.sync` is called. There are multiple drawback of this
-approach.
+While this approach enabled performance optimizations, it introduced significant
```

Comment:
While this approach enables performance optimizations, it introduces significant